### PR TITLE
Force reconnect when receiving aborted status code.

### PR DESCRIPTION
**Pull request by YoEight**
Force reconnect when receiving aborted status code.

### CODE REVIEW

File: db-client-java/src/main/java/com/eventstore/dbclient/GrpcClient.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ protected void pushMsg(Msg msg) {
                 if (error instanceof StatusRuntimeException) {
                     StatusRuntimeException ex = (StatusRuntimeException) error;
 
-                    if (ex.getStatus().getCode().equals(Status.Code.UNAVAILABLE)) {
+                    if (ex.getStatus().getCode().equals(Status.Code.UNAVAILABLE) || ex.getStatus().getCode().equals(Status.Code.ABORTED)) {
```
Comment (oskardudycz) [inline db-client-java/src/main/java/com/eventstore/dbclient/GrpcClient.java:96]: **QUESTION:** Is it possible to write a test simulating this scenario?

Comment (YoEight) [reply db-client-java/src/main/java/com/eventstore/dbclient/GrpcClient.java:96]: A consistent one? Nope. But it happens from time-to-time on the Go client cluster tests.

Comment (oskardudycz) [reply db-client-java/src/main/java/com/eventstore/dbclient/GrpcClient.java:96]: Couldn't it be simulated by killing the node and bringing it back? I know that @George-Payne did some tests like that in NodeJS.

Comment (George-Payne) [reply db-client-java/src/main/java/com/eventstore/dbclient/GrpcClient.java:96]: It tests `UNAVAILABLE`, not sure how you would force an `ABORTED`:
https://github.com/EventStore/EventStore-Client-NodeJS/blob/master/src/__test__/connection/reconnect.test.ts

**Review by oskardudycz — COMMENTED** [review]
Approving it to unblock the client, but tests around that should be tackled as a separate issue.

**Review by oskardudycz — APPROVED** [review]

**Pull request merged**